### PR TITLE
Add option for requiring a match

### LINKED ISSUE #34: The first suggestion should automatically be highlighted when the list of suggestions is displayed

### PR DESCRIPTION
**Pull request by josephschmitt**
Fixes #34

### CODE REVIEW

File: src/angularComponent/ngcOmniboxController.js [hunk 1]
```diff
@@ -63,6 +63,10 @@ export default class NgcOmniboxController {
     }
   }
 
+  $onInit() {
+    this._requireMatch = this.requireMatch === 'true';
```
Comment (onlywei) [inline src/angularComponent/ngcOmniboxController.js]: Why don't we just use `<` binding? I'm not sure I like this private property. We already default to `false`, so `<`'s default behavior matches ours.

Comment (josephschmitt) [inline src/angularComponent/ngcOmniboxController.js]: @onlywei ok

Comment (josephschmitt) [inline src/angularComponent/ngcOmniboxController.js]: Done

File: src/angularComponent/ngcOmniboxController.js [hunk 2]
```diff
@@ -256,7 +256,7 @@ export default class NgcOmniboxController {
    * Un-highlights all suggestions.
    */
   highlightNone() {
-    if (this.isHighlightingDisabled) {
+    if (this.isHighlightingDisabled || this._requireMatch) {
```
Comment (onlywei) [inline src/angularComponent/ngcOmniboxController.js]: Let's test this somehow. I thought that a lot of other methods depend on calling `highlightNone()` to work.

Comment (josephschmitt) [inline src/angularComponent/ngcOmniboxController.js]: Nope, nothing depends on it, they just fire it off blindly.

**Pull request merged**